DevoxxClientMobile/src/main/java/com/devoxx/views/cell/SponsorBadgeCell.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2019, Gluon Software
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without modification, are permitted provided that the
+ * following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation and/or other materials provided
+ *    with the distribution.
+ *
+ * 3. Neither the name of the copyright holder nor the names of its contributors may be used to endorse
+ *    or promote products derived from this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES,
+ * INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.devoxx.views.cell;
+
+import com.devoxx.model.SponsorBadge;
+import com.gluonhq.charm.glisten.visual.MaterialDesignIcon;
+import javafx.scene.Node;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.StackPane;
+
+public class SponsorBadgeCell extends BadgeCell<SponsorBadge> {
+    
+    public SponsorBadgeCell() {
+        getStyleClass().add("sponsor-badge-cell");
+    }
+
+    @Override
+    public void updateItem(SponsorBadge badge, boolean empty) {
+        super.updateItem(badge, empty);
+
+        final Node graphic = MaterialDesignIcon.SYNC_PROBLEM.graphic();
+        final AnchorPane anchorPane = new AnchorPane(graphic);
+        anchorPane.getStyleClass().add("sync-box");
+        AnchorPane.setTopAnchor(graphic, 2.0);
+        AnchorPane.setRightAnchor(graphic, 2.0);
+        if (badge.isSync()) {
+            anchorPane.setVisible(false);
```

Comment:
All that is needed besides `super.updateItem(badge, empty);` is probably this:
`anchorPane.setVisible(! badge.isSync());`